systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -531,6 +531,8 @@ def configure_ipsec(self, obj):
         file.addeq(" pfs=%s" % CsHelper.bool_to_yn(obj['dpd']))
         file.addeq(" keyingtries=2")
         file.addeq(" auto=start")
+        if not obj.has_key('encap'):
```

Comment:
Consider using: if 'encap' not in obj, instead of has_key(). More pythonic that way ;)